e2eAIOK/deltatuner/deltatuner/deltatuner_model.py

```diff
@@ -219,6 +219,12 @@ def from_pretrained(
         from .mapping import DELTATUNER_TYPE_TO_CONFIG_MAPPING, MODEL_TYPE_TO_DELTATUNER_MODEL_MAPPING
         denas_config = kwargs.pop("denas_config", None)
 
+        best_structure_file = os.path.join(model_id, "best_model_structure.txt")
```

Comment:
that means we need to store the "best_model_structure" in the model_id path, how about we can configure the best_model_structure saving path through the arguments

Reply:
When invoking the `from_pretrained` function externally, the input params is `finetune_args.resume_peft`, and then it renamed to `model_id`,  which shows that `model_id` is configurable already.